src/vl-map-search.js

```diff
@@ -94,6 +67,16 @@ class VlMapSearch extends vlElement(HTMLElement) {
       }
     });
   }
+
+  _addSelectChangeListener() {
+    this._selectElement.addEventListener('change', (e) => {
+      if (this._onSelect) {
+        this._onSelect(e.target.value);
```

Comment:
werkt dit nog hetzelfde als in initiële map search? in dat geval OK

Reply:
Dat was alleszins de intentie. Aangezien jullie de enige afnemers zijn, kan dit best door jullie gecontroleerd worden.

Reply:
Misschien te bekijken of we dit nog nodig hebben met deze nieuwe implementatie. Op het eerste zicht halen we gewoon de bbox uit, wat nu via `location` kan bekomen worden (rechtstreeks van vl-select-location). 
Ik denk ook dat toen de aanname was dat dit _resolved_ was, wat hier op het eerste zicht niet meer is.

Reply:
De nieuwe `location` getter is inderdaad een promise. Het value argument blijft normaal ongewijzigd en zou resolved moeten zijn. Ik ben minder voorstander om de value rechtstreeks te gebruiken aangezien dit de response was/is van de achterliggende service. Afnemers zijn bijgevolg momenteel afhankelijk van de response van de achterliggende service. Wanneer deze service wijzigt of we beslissen om een andere service te gebruiken, zijn de afnemers geïmporteerd.

Jullie zijn vermoedelijk de enige afnemers van de callback, aangezien die op jullie vraag geïntroduceerd werd? We zouden nu kunnen beslissen om de value aan te passen naar de resolved location bounding box. De breaking changes documenteren we dan bij release.

Reply:
Prima wat mij betreft. Vermoedelijk hebben wij zelfs die callback ook niet meer nodig als we rechtstreeks met `VlSelectLocation` werken.

@ZhaodiFengVlaanderen aangezien jij hier vooral aan gewerkt hebt, graag bevestiging.